modules/firemission/functions/FIREMIS/fn_FIREMIS_RegisterForwardObserver.sqf

```diff
@@ -1,7 +1,6 @@
 #include "script_component.hpp"
 
-if (isServer) then
-{
+if (!isMultiplayer || !hasInterface) then { // If this is the server or headless client
```

Comment:
Have you checked that this doesn't make it impossible to test in the editor? I tried using !hasInterface before, but then couldn't test it, because technically my PC was the server and also had an interface.

Reply:
it's an OR, so if not multiplayer, it works

Reply:
I wasn't sure if that returned true if you were testing it in multiplayer mode, which is what I do?

Reply:
you can't use the HC module properly in multiplayer editor, use SP editor...

Reply:
to test properly you need to host server, HC, and your client arma instance

Reply:
Alrighty, but I mean this check will stop being able to test AI FOs in the firemission module via the MP editor.

Reply:
what is the benefit of testing in the MP editor? Many modules wont work properly in it, including HC

Reply:
It's just what I do and I don't like change :'(

Reply:
As long as it's testable in SP that's fine.